### PR TITLE
[CM-42] Group related UI Test Cases #trivial

### PR DESCRIPTION
**Pull request by archit2602**
## Summary
Group related UI Test Cases
1. Send Text message
2. Send Image
3. Send Location
4. Send Contact

### CODE REVIEW

File: Demo/ApplozicSwiftDemoUITests/ApplozicSwiftGroupSendMessageUITest.swift [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+            if(alerts.buttons[AppPermission.AlertButton.allow].exists){
+                alerts.buttons[AppPermission.AlertButton.allow].tap();
+            }
+            return true;
+        }
+        
+        // UI tests must launch the application that they test. Doing this in setup will make sure it happens for each test method.
+        XCUIApplication().launch()
+        
+        sleep(5)
```
Comment (mukeshthawani) [inline Demo/ApplozicSwiftDemoUITests/ApplozicSwiftGroupSendMessageUITest.swift]: We can replace `sleep` with `waitForExpectations` in all the test cases. This way tests will run fast and, if the element is not present then that test will fail with a message. You can add [this extension](https://stackoverflow.com/a/41610891/3637751).

File: Demo/ApplozicSwiftDemoUITests/ApplozicSwiftGroupSendMessageUITest.swift [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+        app.buttons[InAppButton.ConversationScreen.openPhotos].tap() // Click on photo button
+        addUIInterruptionMonitor(withDescription: AppPermission.AlertMessage.accessPhoto ) { (alerts) -> Bool in
+            if(alerts.buttons[AppPermission.AlertButton.ok].exists){
+                alerts.buttons[AppPermission.AlertButton.ok].tap();
+            }
+            return true;
+        }
+        app.tap()
+        sleep(2)
+        app.collectionViews.children(matching: .cell).element(boundBy: 0).children(matching: .other).element.tap()
```
Comment (mukeshthawani) [inline Demo/ApplozicSwiftDemoUITests/ApplozicSwiftGroupSendMessageUITest.swift]: It is not clear, what's happening here. Maybe we can name the element and then tap.

File: Demo/ApplozicSwiftDemo/ViewController.swift [hunk 3]
```diff
@@ -35,6 +35,17 @@ class ViewController: UIViewController {
...
+        let contact1  = ALContact()
+        let contact2 = ALContact()
+        
+        contact1.userId = "IoSDemoContact1"
+        contact1.displayName = "IoS Demo Contact 1"
+        contact2.userId = "IoSDemoContact2"
+        contact2.displayName = "IoS Demo Contact 2"
+        
+        let contactService = ALContactService()
+        contactService.addList(ofContacts: [contact1 , contact2])
```
[the remark this answers is not in the document]
Comment (archit2602) [reply Demo/ApplozicSwiftDemo/ViewController.swift]: done

File: Demo/ApplozicSwiftDemoUITests/ApplozicSwiftAudioRecordingUITests.swift [hunk 4]
```diff
@@ -169,13 +169,15 @@ class ApplozicSwiftAudioRecordingUITest: XCTestCase {
     
     private func beforeTest_EnterConversation()  -> (XCUIApplication, XCUIElement, XCUIElement){
         let app = XCUIApplication()
+        let path = Bundle(for: ApplozicSwiftAudioRecordingUITest.self).url(forResource: "Info", withExtension: "plist")
+               let dict = NSDictionary(contentsOf: path!) as? [String: Any]
```
[the remark this answers is not in the document]
Comment (archit2602) [reply Demo/ApplozicSwiftDemoUITests/ApplozicSwiftAudioRecordingUITests.swift]: done

File: Demo/ApplozicSwiftDemoUITests/ApplozicSwiftAudioRecordingUITests.swift [hunk 5]
```diff
@@ -169,13 +169,15 @@ class ApplozicSwiftAudioRecordingUITest: XCTestCase {
...
         app.buttons["Launch Chat"].tap()
         sleep(2)
         app.navigationBars["My Chats"].buttons["fill 214"].tap()
         sleep(3)
         let searchField = app.searchFields["Search"]
         searchField.tap()
-        searchField.typeText("user2ToTestAudioRecord")
+        searchField.typeText(dict!["GroupMember1"] as! String)
```
[the remark this answers is not in the document]
Comment (archit2602) [reply Demo/ApplozicSwiftDemoUITests/ApplozicSwiftAudioRecordingUITests.swift]: done

File: Demo/ApplozicSwiftDemoUITests/ApplozicSwiftGroupSendMessageUITest.swift (hunk 1 above)
Comment (archit2602) [reply Demo/ApplozicSwiftDemoUITests/ApplozicSwiftGroupSendMessageUITest.swift]: done

File: Demo/ApplozicSwiftDemoUITests/ApplozicSwiftGroupSendMessageUITest.swift (hunk 2 above)
Comment (archit2602) [reply Demo/ApplozicSwiftDemoUITests/ApplozicSwiftGroupSendMessageUITest.swift]: done

**Pull request merged**